app/kubernetes/views/deploy/deploy.html

```diff
@@ -38,7 +38,7 @@
                       ng-model="ctrl.formValues.Namespace"
                       ng-options="namespace.Name as namespace.Name for namespace in ctrl.namespaces"
                     ></select>
-                    <span ng-if="ctrl.formValues.namespace_toggle">Namespaces specified in the manifest will be used</span>
+                    <span ng-if="ctrl.formValues.namespace_toggle" class="small text-muted pt-[7px]">Namespaces specified in the manifest will be used</span>
```

Comment:
Will pt-2 look ok? (8px)

Reply:
label has padding 7 px in global css and to make it aligned with the label, i used this.